Sources/LanguageServerProtocol/SupportTypes/TestItem.swift

```diff
@@ -67,8 +67,10 @@ public struct TestItem: ResponseType, Equatable {
     children: [TestItem],
     tags: [TestTag]
   ) {
-    self.id = id
-    self.label = label
+    // Filtering backticks is temporary until we can use
+    // https://github.com/apple/swift-syntax/pull/2576
+    self.id = id.filter { $0 != "`" }
+    self.label = id == label ? self.id : label
```

Comment:
I would prefer to only do the filtering on the function name. Once we start including the file’s name in the ID (#1152), we shouldn’t remove the backtick from eg. the filename.

Reply:
@grynspan Can you specify running a swift-testing test via --filter with an ID that has the file's name + line + column in the ID, a la #1152?

My main requirement for these IDs is that I can take the IDs that sourcekit-lsp and pass them directly to swift-testing without converting them in any way.